File: tests/testdata/codegen/using-dashes-pp/dotnet/using-dashes.cs
using System.Collections.Generic;
using System.Linq;
using Codeinfra;
using UsingDashes = Codeinfra.UsingDashes;

return await Deployment.RunAsync(() => 
{
    var main = new UsingDashes.Dash("main", new()
    {
        Stack = "dev",
    });

});

File: tests/testdata/codegen/using-dashes-pp/go/using-dashes.go
package main

import (
	usingdashes "example.com/codeinfra-using-dashes/sdk/go/using-dashes"
	"github.com/khulnasoft/codeinfra/sdk/v3/go/codeinfra"
)

func main() {
	codeinfra.Run(func(ctx *codeinfra.Context) error {
		_, err := usingdashes.NewDash(ctx, "main", &usingdashes.DashArgs{
			Stack: codeinfra.String("dev"),
		})
		if err != nil {
			return err
		}
		return nil
	})
}
